php/content.md

```diff
@@ -168,7 +168,6 @@ FROM %%IMAGE%%:7.4-fpm-alpine
 
 # Use the default production configuration
 RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
-
-# Override with custom opcache settings
-COPY config/opcache.ini $PHP_INI_DIR/conf.d/
```

Comment:
@yosifkit Why was this removed?

The `conf.d`-directory is a custom scan-dir, so it is **not duplicating common PHP documentation**.
https://github.com/docker-library/php/blob/master/7.4/alpine3.12/fpm/Dockerfile#L125

Without this info it is kind of hard to know were to put custom config files, that's why the info was there.

Reply:
See a few lines up:

> The default config can be customized by copying configuration files into the `$PHP_INI_DIR/conf.d/` directory.

Reply:
Thanks for the hint! (I must be blind :man_facepalming:)